### PR TITLE
Use relevant global object to obtain the Window

### PR DESCRIPTION
**Pull request by npm1**
The Event target itself is not a Window. The Window is obtained from the target's relevant global object. This PR fixes https://github.com/WICG/event-timing/issues/32

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -83,6 +83,8 @@ urlPrefix: https://w3c.github.io/uievents/; spec: UIEVENTS;
     type: dfn; url: #event-type-compositionstart; text: compositionstart;
     type: dfn; url: #event-type-compositionupdate; text: compositionupdate;
     type: dfn; url: #event-type-compositionend; text: compositionend;
+urlPrefix: https://heycam.github.io/webidl/; spec: WEBIDL;
+    type: dfn; url: #implements; text: implement;
```
Comment (domenic) [inline index.bs]: This shouldn't be necessary; you can use `lt="implements"`, and we should add an alias in Web IDL.

File: index.bs [hunk 2]
```diff
@@ -391,10 +395,11 @@ This provides user agents with the flexibility to ignore input which never block
 <div algorithm="report outside of event dispatch">
     To <em>create an event timing entry</em> outside of the <a>event dispatch algorithm</a>, the user agent must run the following steps:
 
-    1. Let <var>event</var> be the event being measured and let <var>target</var> be the target {{Window}}.
+    1. Let <var>event</var> be the event being measured and let <var>window</var> be the <a>relevant global object</a> of the target.
+    1. Assert that <var>window</var> <a>implements</a> {{Window}}.
```
Comment (domenic) [inline index.bs]:
```suggestion
    1. Assert: <var>window</var> <a>implements</a> {{Window}}.
```

File: index.bs (hunk 1 above)
Comment (npm1) [reply index.bs]: There is already <dfn export>implements</dfn> in Web IDL. I don't understand why it does not pick it up without this.

Comment (npm1) [reply index.bs]: To clarify, I see `<dfn export>implements</dfn>` but `<a>implements</a>` does not work. In comparison, something like `<a>partial interface mixin</a>` works right away, presumably due to the corresponding dfn export?

Comment (domenic) [reply index.bs]: That is bizarre. Have you done `bikeshed update` recently / are you using the web service?

Comment (npm1) [reply index.bs]: Ahh that was the problem! I think I had not updated in a while.

**Pull request merged**